### PR TITLE
Grtlv 253

### PR DESCRIPTION
**Pull request by bobby-didcoding**
Learn to Export HCSAT (logged out).

You will need to change FEATURE_DEA_V2 = True in your env for this to work. 

To test, visit http://greatcms.trade.great:8020/learn/categories/market-research/evaluate-opportunities/opportunity-right-you/ as a logged out user.

You will find the HCSAT form at the bottom of the page.
Screenshot 2024-08-14 at 11 24 36

Go ahead and submit your feedback, you should then see the following:
Screenshot 2024-08-14 at 11 24 49

Submitting the second half of the form will complete the HCSAT form and you'll see this:
Screenshot 2024-08-14 at 11 25 01

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GRTLV-253
- [x] Jira ticket has the correct status.
- [x] A clear/description pull request messaged added.

### Reviewing help

- [x] Explains how to test locally, including how to set up appropriate data
- [x] Includes screenshot(s) - ideally before and after, but at least after

### Merging

- [x] This PR can be merged by reviewers.

### CODE REVIEW

File: core/geolocation_data/GeoLite2-Country.mmdb
Comment (timothyPatterson) [inline core/geolocation_data/GeoLite2-Country.mmdb:1]: Failing test is because this file is being deleted.

File: core/models.py [hunk 1]
```diff
@@ -1205,12 +1238,60 @@ def _get_backlink_title(self, backlink_path):
...
+    def get_csat_model(self):
+        """Import the learn CSAT model here to avoid import conflicts"""
         from learn.models import CsatUserFeedback
 
+        return CsatUserFeedback
+
+    def get_success_url(self, request):
+        return request.path
+
+    def js_enabled(self, request):
```
Comment (timothyPatterson) [inline core/models.py:1251]: This could maybe be simplified to `return 'js_enabled' in request.get_full_path()`

**Review by timothyPatterson — APPROVED** [review]

File: core/models.py (hunk 1 above)
Comment (bobby-didcoding) [reply core/models.py:1251]: Thanks @timothyPatterson. Change is in the most recent commit

File: core/geolocation_data/GeoLite2-Country.mmdb
Comment (bobby-didcoding) [reply core/geolocation_data/GeoLite2-Country.mmdb:1]: I don't understand how this got deleted but it has been added in the most recent commit :)

**Pull request merged**